src/pivot-table/components/cells/DimensionValue/utils/get-justify-content.ts

```diff
@@ -0,0 +1,12 @@
+import type { ExtendedDimensionInfo } from "../../../../../types/QIX";
+import type { Cell, Flags, LayoutService } from "../../../../../types/types";
+import resolveTextAlign from "../../utils/resolve-text-align";
+
+const getJustifyContent = (cell: Cell, layoutService: LayoutService, isLeftColumn: boolean, flags: Flags) => {
```

Comment:
The idea behind `getJustifyContent` is that it will be used both for the text aligment feature and the URL image feature to position content.